src/components/withBlockViewportScroll/index.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import Keyboard from '../../libs/NativeWebKeyboard';
+import getComponentDisplayName from '../../libs/getComponentDisplayName';
+
+export default function(WrappedComponent) {
+    function WithBlockViewportScroll(props) {
+        const optimalScrollY = useRef(0);
+        const keybShowOff = useRef(() => { });
+        const keybHideOff = useRef(() => { });
```

Comment:
no shortened variable names, and keep meaningful names

Reply:
renamed for more clarity